src/src/hooks/SelfServiceApi.js

```diff
@@ -27,6 +30,8 @@ export function useSelfServiceRequest(errorParams) {
       url = urlSegments[0];
     }
 
+    track('selfservice', `${method}::${url}`, '1')
```

Comment:
'1' seem like a strange EventName, but I am not an expert on Matomo at all.
If we are trying to record an EventValue, then maybe we should supply an empty name?

This is just me guessing blindly from documentation and not so much practical playing around